deps/wazuh_testing/wazuh_testing/modules/fim/event_monitor.py

```diff
@@ -97,6 +97,48 @@ def callback_disk_quota_limit_reached(line):
         return match.group(2)
 
 
+def callback_detect_file_added_event(line):
+    """ Callback that detects if a line in a log is a file added event.
+    Args:

```

Comment:
```suggestion
    
    Args:

```

Reply:
Add also the `Returns` clause.

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/3669/commits/0c6e5d9a13ebda78ada36a7c9c8b18a9617ad6a8